runtime/src/main/java/org/corfudb/runtime/view/LayoutView.java

```diff
@@ -325,4 +326,22 @@ public void committed(long epoch, Layout layout, boolean force)
             log.debug("committed: Successful responses={}, timeouts={}", responses, timeouts);
         }
     }
+
+    /**
+     * Bootstraps the layout server of the specified node.
+     * If already bootstrapped, it completes silently.
+     *
+     * @param endpoint Endpoint to bootstrap.
+     * @param layout   Layout to bootstrap with.
+     */
+    public void bootstrapLayoutServer(@Nonnull String endpoint, @Nonnull Layout layout) {
+        try {
+            CFUtils.getUninterruptibly(getRuntimeLayout(layout)
```

Comment:
Can you please use async computation. Just return completable future. Then compose this future with a future provided by `bootstrapManagementServer()` method.
If you do that you don't need to have a boilerplate block of code `try, catch`, you just need to handle an exception in one place